### PR TITLE
Added e2e test for cluster autoscaler that verifies host ports.

### PR DESCRIPTION
**Pull request by jszczepkowski**
[Analytics]()

Added e2e test for cluster autoscaler that verifies host ports.

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @mwielgus @fgrzadkowski

File: test/e2e/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -65,11 +65,38 @@ var _ = framework.KubeDescribe("Cluster size autoscaling [Feature:ClusterSizeAut
 		// Verify, that cluster size is increased
 		framework.ExpectNoError(framework.WaitForClusterSize(f.Client, nodeCount+1, scaleTimeout))
 		framework.ExpectNoError(framework.DeleteRC(f.Client, f.Namespace.Name, "memory-reservation"))
-		// TODO(jsz): Enable code bellow when scale down is implemented.
-		// framework.ExpectNoError(framework.WaitForClusterSize(f.Client, nodeCount, scaleDownTimeout))
+		// TODO(jsz): Disable the line bellow when scale down is implemented.
```
Comment (piosz) [inline test/e2e/cluster_size_autoscaling.go]: FYI @mwielgus

**Comment by piosz** [comment]
LGTM

**Comment by jszczepkowski** [comment]
This PR should be merged after https://github.com/kubernetes/test-infra/pull/57.

File: test/e2e/cluster_size_autoscaling.go [hunk 2]
```diff
@@ -65,11 +65,38 @@ var _ = framework.KubeDescribe("Cluster size autoscaling [Feature:ClusterSizeAut
...
+		framework.ExpectNoError(ResizeGroup(int32(nodeCount)))
+		framework.ExpectNoError(framework.WaitForClusterSize(f.Client, nodeCount, scaleTimeout))
+
+		By("Handling node port pods")
+		CreateHostPortPods(f, "host-port", nodeCount+2, false)
+		framework.ExpectNoError(framework.WaitForClusterSize(f.Client, nodeCount+2, scaleTimeout))
+		framework.ExpectNoError(framework.DeleteRC(f.Client, f.Namespace.Name, "host-port"))
+		// TODO(jsz): Disable the line bellow when scale down is implemented.
```
Comment (jszczepkowski) [inline test/e2e/cluster_size_autoscaling.go]: FYI @mwielgus

**Pull request merged**